portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,14 +26,11 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  /** A temporary set of comments that we will hardcode. */
+  /** A simple list of text-only comments. */
   ArrayList<String> comments = new ArrayList<String>();
```

Comment:
Nit: This should be a private List<String>

Reply:
Fixed.